app/assets/javascripts/navbar.js

```diff
@@ -0,0 +1,167 @@
+                  {" "}
+                  <Link to="/taskTypes">Task Types</Link>
+                </Menu.Item>
+                <Menu.Item key="/scripts">
+                  {" "}
+                  <Link to="/scripts">Scripts</Link>
+                </Menu.Item>
+              </SubMenu>,
+              <SubMenu
+                key="sub2"
+                title={
+                  <span>
+                    <Icon type="line-chart" />Statistics
+                  </span>
+                }
+              >
+                <Menu.Item key="/tasks/overview">
+                  <Link to="/tasks/overview">Overview</Link>
+                </Menu.Item>
+                <Menu.Item key="/statistics">
+                  <Link to="/statistics">Weekly</Link>
+                </Menu.Item>
+              </SubMenu>,
+              <SubMenu
+                key="sub3"
+                title={
+                  <span>
+                    <Icon type="medicine-box" />Help
+                  </span>
+                }
+              >
+                <Menu.Item key="/help/faq">
+                  <a target="_blank" href="/help/faq">
+                    FAQ
+                  </a>
+                </Menu.Item>
+                <Menu.Item key="11">
+                  <a target="_blank" href="/assets/docs/frontend-api/index.html">
+                    Frontend API Documentation
+                  </a>
+                </Menu.Item>
+                <Menu.Item key="/help/keyboardshortcuts">
+                  <a target="_blank" href="/help/keyboardshortcuts">
+                    keyboard Shortcuts
+                  </a>
+                </Menu.Item>
+              </SubMenu>,
+              <Menu.Item key="13">
+                <a href="https://discuss.webknossos.org" target="_blank" rel="noopener noreferrer">
+                  <Icon type="notification" />
+                  Discussion Board
+                </a>
+              </Menu.Item>,
+              <SubMenu
+                key="sub4"
+                className="pull-right"
+                title={
+                  <span>
+                    <Icon type="user" />
+                    {`${this.props.activeUser.firstName} ${this.props.activeUser.lastName}`}
+                  </span>
+                }
+              >
+                <Menu.Item key="resetpassword">
+                  <Link to="/reset">Reset Password</Link>
+                </Menu.Item>
+                <Menu.Item key="logout">
+                  <Link
+                    to="/"
+                    onClick={() => {
+                      Request.receiveJSON("/api/logout").then(() => window.location.reload());
+                    }}
+                  >
+                    Logout
+                  </Link>
+                </Menu.Item>
+              </SubMenu>,
+            ]
+          ) : (
+            <Menu.Item key="login">
+              <LoginView layout="inline" />
+            </Menu.Item>
+          )}
```

Comment:
I think here could be also a link to `/reset`

Reply:
The `<LoginView>`component already includes a link to reset the password.